src/main/lua/exasolvs/query_renderer.lua

```diff
@@ -83,21 +85,48 @@ function M.new (query)
         append('"')
     end
 
+    local function append_function_argument_list(arguments)
+        append("(")
+        if (arguments) then
+            for i = 1, #arguments do
+                comma(i)
+                append_expression(arguments[i])
+            end
+        end
+        append(")")
+    end
+    
+    local function append_arithmetic_function(left, operator, right)
+        append_expression(left)
+        append(" ")
+        append(operator)
+        append(" ")
+        append_expression(right)
+    end
+
     local function append_scalar_function(scalar_function)
         local function_name = string.upper(scalar_function.name)
         if M.supported_scalar_functions[function_name] then
-            append(function_name)
-            if function_name ~= "CURRENT_USER" and function_name ~= "SYSDATE" and function_name ~= "CURRENT_SCHEMA"
-                and function_name ~= "CURRENT_SESSION" and function_name ~= "CURRENT_STATEMENT" then
-                append("(")
+            if function_name == "CURRENT_USER" or function_name == "SYSDATE" or function_name == "CURRENT_SCHEMA"
+                or function_name == "CURRENT_SESSION" or function_name == "CURRENT_STATEMENT" then
```

Comment:
Extract these five special functions into a list (e.g. `no_arg_functions`?) and check if `function_name` is in this list. That would make the intention clearer.

Reply:
[IPROC]((https://docs.exasol.com/sql_references/functions/alphabeticallistfunctions/iproc.htm) tells you on which cluster node a row resides. We have two rows on a single-node cluster in this test, so the node index is zero for every row.

Reply:
Extracted method with a speaking name.